packages/notification/src/components/GenericNotification.tsx

```diff
@@ -10,7 +10,7 @@ import successIcon from "@kaizen/component-library/icons/success.icon.svg"
 import styles from "./GenericNotification.module.scss"
 
 export type NotificationType =
-  | "affirmative"
+  | "positive"
```

Comment:
Just making sure you're planning to release this as a breaking change?

Reply:
@dougmacknz Yes, definitely. I was actually just looking up if it suffices to do it in the final commit/merge message or whether it has to be in the PR title.

Reply:
You're right, it just needs to be in the final commit/merge message.